plugins/VerboseLogPlugin/index.js

```diff
@@ -25,14 +28,20 @@ class VerboseLogPlugin {
 			}
 		};
 
+		let isStart = false;
```

Comment:
If the setTimeout delay set zero, I think isStart is unnecessary.

Reply:
OK